modules/@apostrophecms/rich-text-widget/ui/apos/components/ApostropheTiptapLink.vue

```diff
@@ -28,6 +30,7 @@
         <AposContextMenuDialog
           menu-placement="bottom-start"
           v-if="active"
+          class-list="apos-theme-light"
```

Comment:
I'm not sure how I feel about this given that a whole other third theme could be applied at the top sometime later.

Reply:
This feels weird to me too but I defer to the frontend experts.

Reply:
:stinky: